Playwright tests and minor design

shech2 opened the pull request:
check this out.

idoRosen25 committed:
add basic logic for invitation email

idoRosen25 committed:
add basic logic for invitation email

idoRosen25 committed:
update readme.md

shech2 committed:
added a few minor improvments tell me if you like

git push

exot
exit

shech2 committed:
added a few minor improvments tell me if you like

git push

exot
exit

idoRosen25 commented:
Not sure I understand what is done here. Seems it also includes my invitation email changes when it shouldn't have them.
Rebase and make sure only the new changes exist. Otherwise mark as draft until I merge my changes

shech2 committed:
Added tests

shech2 committed:
Updated readme

shech2 committed:
Login and signup tests, not finished yet

shech2 commented:
> Not sure I understand what is done here. Seems it also includes my invitation email changes when it shouldn't have them. Rebase and make sure only the new changes exist. Otherwise mark as draft until I merge my changes

i added a plus button on header and put invite page inside the layout and added tests

shech2 committed:
Updated README

shech2 committed:
Cleaver tests

shech2 committed:
a little fix

shech2 committed:
Modyfied playwright config

shech2 committed:
Modyfied playwright config

shech2 committed:
added workflow file

shech2 committed:
fix yaml

shech2 committed:
.

shech2 committed:
try to make it work

shech2 committed:
SAB

shech2 committed:
SAB

shech2 committed:
SAB

shech2 committed:
Trying something new

shech2 committed:
SAB

shech2 committed:
SAB

shech2 committed:
SAB

shech2 committed:
Last Try

shech2 committed:
SAB

shech2 committed:
SAB

shech2 committed:
Last chance

shech2 committed:
update

shech2 commented:
There is a problem with the env... but i dont know why

shech2 committed:
checking something new

shech2 committed:
SAB

shech2 committed:
should work

shech2 committed:
another try

shech2 committed:
fix

shech2 committed:
trying now

shech2 committed:
Have to work now

shech2 committed:
Better tests

shech2 committed:
tests for tasks

idoRosen25 commented on package.json:
```diff
@@ -12,6 +12,7 @@
     "prisma:generate": "npx prisma generate",
     "prisma:validate": "npx prisma validate",
     "migrate:postgres": "prisma migrate dev",
+    "migrate:dev": "dotenv -e .env.local -- npx prisma migrate dev",
```
Why is this needed?

shech2 replied:
I tried to load the env to run the workflows , it doesn't work , we need to add secrets to GitHub in order for that to work

shech2 replied:
Removed this line

idoRosen25 commented on package.json:
```diff
@@ -51,6 +54,8 @@
     "@types/react": "^18",
     "@types/react-dom": "^18",
     "autoprefixer": "^10.0.1",
+    "dotenv": "^16.4.1",
```
Why do we need this?

shech2 replied:
SAO

idoRosen25 commented on tests-examples/demo-todo-app.spec.ts:
```diff
@@ -0,0 +1,437 @@
+import { test, expect, type Page } from '@playwright/test';
+
```
Is this file needed? Seems like an example and not a true test

shech2 replied:
Nope i will delete it

idoRosen25 reviewed (changes requested):
Seems fine on the whole. Some points I'm not sure what they mean

shech2 committed:
Deleted example tests

shech2 committed:
minor changes

shech2 committed:
Added logo to login page

shech2 committed:
Better practice

shech2 committed:
added on deployment

shech2 committed:
changed names

shech2 committed:
Better now

shech2 committed:
SAB

shech2 committed:
fix

shech2 committed:
fix

shech2 committed:
dependency on push

shech2 committed:
SAB

shech2 committed:
added a new cmd

idoRosen25 committed:
add basic logic for invitation email (#17)

* add basic logic for invitation email

* update readme.md

* fix navigations and invite email

shech2 commented:
Mark as draft too many commits, i will do it cleaner later and delete this pull..
